analysers/analyser_osmosis_highway_area_access.py

```diff
@@ -94,25 +94,25 @@
 
 sql31 = """
 SELECT
-  highways.id,
+  ways.id,
   barrier.id,
   ST_AsText(barrier.geom)
 FROM
-  highways
-  JOIN nodes AS barrier ON
+  {0}highways as ways
```

Comment:
You can also just keep the name of the table ?
```suggestion
  {0}highways as highways
```

Reply:
True, I've updated the PR